packages/wonder-blocks-tooltip/components/tooltip-anchor.js

```diff
@@ -252,7 +260,9 @@ export default class TooltipAnchor extends React.Component<Props, State>
             this._timeoutID = setTimeout(() => {
                 this._timeoutID = null;
                 this._setActiveState(active, true);
+                this.props.onTimeoutChanged(null);
             }, delay);
+            this.props.onTimeoutChanged(this._timeoutID);
```

Comment:
What happens if `instant` is `true`?  Do we need to call `this.props.onTimeoutChanged` in that case as well?

Reply:
Thanks for the catch, I don't believe we would need to actually, but I fixed it up in the latest commit regardless.